test/admin-hpa/features/step_definitions/data_sync_steps.rb

```diff
@@ -4,8 +4,11 @@
 end
 
 When(/^the data sync function has run$/) do
-  response = FunctionsHelper.resync_all(true)
-  expect(response.code).to eql 202
+  if @check_code
+    (wait_until(120, 2) {SqlDbHelper.get_pupil_check_metadata(@check_code)['checkStatus_id'] == 2}) if @no_answers.nil?
```

Comment:
might be a good idea to make the 2 minute wait time configurable, just in case

Reply:
@GuyHarwood thanks good point, updated now.